### PR TITLE
fixing imports

### PR DESCRIPTION
**Pull request by ue71603**
xmllint has some problems with multiple imports with the same namespace. Fixes a problem in OJP.
Place where we locally fixed it in OJP: https://github.com/VDVde/OJP/pull/347

### CODE REVIEW

File: xsd/siri_facilityMonitoring_service.xsd [hunk 1]
```diff
@@ -74,13 +74,13 @@ Rail transport, Roads and road transport
...
  <xsd:import namespace="http://www.ifopt.org.uk/ifopt" schemaLocation="ifopt/ifopt_allStopPlace.xsd"/>
- <xsd:import namespace="http://www.ifopt.org.uk/ifopt" schemaLocation="ifopt/ifopt_stop.xsd"/>
+ <!-- <xsd:import namespace="http://www.ifopt.org.uk/ifopt" schemaLocation="ifopt/ifopt_stop.xsd"/> -->
  <!--.....See facility for the actual dependencies...-->
  <!-- ===Global import of all ACSB namespace elements needed to work around XERCES limitation This needs to be after ifopt=====-->
  <xsd:import namespace="http://www.ifopt.org.uk/acsb" schemaLocation="acsb/acsb_all.xsd"/>
  <!--.....These are the actual dependencies...-->
- <xsd:import namespace="http://www.ifopt.org.uk/acsb" schemaLocation="acsb/acsb_passengerMobility.xsd"/>
- <xsd:import namespace="http://www.ifopt.org.uk/acsb" schemaLocation="acsb/acsb_accessibility.xsd"/>
+ <!-- <xsd:import namespace="http://www.ifopt.org.uk/acsb" schemaLocation="acsb/acsb_passengerMobility.xsd"/>
```
Comment (skinkie) [inline xsd/siri_facilityMonitoring_service.xsd:82]: I suggest not using comments in pull requests.

**Comment by Aurige** [comment]
Ok for me, but is it expected to also have this huge reformatting of some of the files ? That probably means that SIRI also requires that xmllint pretty action that was applied on OJP and NeTEx

**Comment by ue71603** [comment]
@Aurige At some point I think we will need to do the reformatting. If we wish to use the same github actions.
Best therfore to minimize the number of PR in github.

**Comment by ue71603** [comment]
or the current rule, if it exists needs to be incorporated into the action.

**Comment by skinkie** [comment]
One step at a time. Migrating to github actions would be something separate.

**Review by Aurige — APPROVED** [review]

**Review by AndreasAtSBB — APPROVED** [review]
Removed some imports, removed prefixes ojp: and siri:.
Schould now compile.

**Review by sgrossberndt — CHANGES_REQUESTED** [review]
This PR is broken as @ue71603 simply backported the changes done to OJP to SIRI, but in https://github.com/VDVde/OJP/commit/241cf4b7d812dad4251da4116376c6c3fe532978, where SIRI was updated to 2.1 he changed the file names of the ifopt files and removed the version numbers there, while they still have them here.

Also as mentioned before the imports should not be commented out but adapted.

**Review by sgrossberndt — APPROVED** [review]

**Comment by sgrossberndt** [comment]
Is the Travis integration broken for SIRI?

**Comment by skinkie** [comment]
> Is the Travis integration broken for SIRI?

I think everything was broken due to credits etc.

**Pull request merged**